### PR TITLE
Branch1 resume information

### PR DESCRIPTION
**Pull request by asavienko**

### CODE REVIEW

File: api/services/getAuthToken/initBrowser.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const puppeteer = require("puppeteer");
-const { browserHeadless } = require("../../toDellete/constants/constants");
-
+const browserHeadless = require("../../toDellete/constants/constants")
```
Comment (bohdanSnappy) [inline api/services/getAuthToken/initBrowser.js]:
```suggestion
const {browserHeadless} = require("../../toDellete/constants/constants")
```

File: api/services/parseResumeDetails/parseDetails.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  collectionReports,
+  collectionResumes
+}) => {
+  function timeout(ms) {
+    return new Promise(resolve => setTimeout(resolve, ms));
+  }
+  const parseEachResume = async (previousResume, skip) => {
+    const [{ _id }] =
+      previousResume ||
+      (await collectionResumes
```
Comment (bohdanSnappy) [inline api/services/parseResumeDetails/parseDetails.js]: move to separate function

File: api/services/parseResumeDetails/parseDetails.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+          { $skip: skip },
+          { $limit: 1 }
+        ])
+        .toArray());
+    const url = `https://employer-api.rabota.ua/resume/${_id}`;
+    console.log(`id in url: ${_id}`);
+    try {
+      const response = await fetch(url, options);
+      const { status: responseStatus } = response;
+      console.log(responseStatus);
```
Comment (bohdanSnappy) [inline api/services/parseResumeDetails/parseDetails.js]: remove all conloge.log

File: api/services/parseResumeDetails/parseDetails.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+            error: `Error status: ${responseStatus}`,
+            resumeId: _id,
+            time: new Date()
+          },
+          reportId,
+          collectionReports
+        );
+        await timeout(30000);
+        parseEachResume([{ _id }], skip);
+      } else if (responseStatus === 200) {
```
Comment (bohdanSnappy) [inline api/services/parseResumeDetails/parseDetails.js]: move this condition to up

File: api/services/parseResumeDetails/parseDetails.js [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+      console.log(responseStatus);
+      if (responseStatus === 204) {
+        const report = await saveErrorStatus({
+          responseStatus,
+          collectionResumes,
+          _id
+        });
+        await saveReport(report, reportId, collectionReports);
+        parseEachResume(null, skip);
+      } else if (responseStatus !== 200) {
```
Comment (bohdanSnappy) [inline api/services/parseResumeDetails/parseDetails.js]: remove this condition

File: api/services/parseResumeDetails/parseDetails.js [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+            resumeId: _id,
+            time: new Date()
+          },
+          reportId,
+          collectionReports
+        );
+        await timeout(30000);
+        parseEachResume([{ _id }], skip);
+      } else if (responseStatus === 200) {
+        const json = await response.json();
```
Comment (bohdanSnappy) [inline api/services/parseResumeDetails/parseDetails.js]:
```suggestion
        const data = await response.json();
```

File: api/services/parseResumeDetails/parseDetails.js [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+        const report = await saveToDb({
+          data: json,
```
Comment (bohdanSnappy) [inline api/services/parseResumeDetails/parseDetails.js]:
```suggestion
          data,
```

File: api/services/parseResumeDetails/saveToDb.js [hunk 8]
```diff
@@ -0,0 +1,24 @@
+const saveToDb = async ({ data, collectionResumes }) => {
+  const { resumeId: _id } = data;
+  delete data.resumeId;
```
Comment (bohdanSnappy) [inline api/services/parseResumeDetails/saveToDb.js]: remove where you get

File: api/services/getAuthToken/initBrowser.js (hunk 1 above)
Comment (asavienko) [reply api/services/getAuthToken/initBrowser.js]: edited

File: api/services/parseResumeDetails/parseDetails.js (hunk 2 above)
Comment (asavienko) [reply api/services/parseResumeDetails/parseDetails.js]: moved

File: api/services/parseResumeDetails/parseDetails.js (hunk 4 above)
Comment (asavienko) [reply api/services/parseResumeDetails/parseDetails.js]: has moved

File: api/services/parseResumeDetails/parseDetails.js (hunk 5 above)
Comment (asavienko) [reply api/services/parseResumeDetails/parseDetails.js]: removed

File: api/services/parseResumeDetails/parseDetails.js (hunk 6 above)
Comment (asavienko) [reply api/services/parseResumeDetails/parseDetails.js]: edited

File: api/services/parseResumeDetails/parseDetails.js (hunk 7 above)
Comment (asavienko) [reply api/services/parseResumeDetails/parseDetails.js]: edited

File: api/services/parseResumeDetails/saveToDb.js (hunk 8 above)
Comment (asavienko) [reply api/services/parseResumeDetails/saveToDb.js]: edited

**Pull request merged**